docs/source/known_issues.rst

```diff
@@ -8,3 +8,6 @@ There are several known issues with the current version of Nethermind.
  
  RocksDB on macOS
    RocksDB library does not always load properly on macOS. One (hacky) workaround is to install the latest version of RocksDB by running brew install rocksdb.
+
+ Skipping consensus issues blocks
+   We do our best in Nethermind not to have consensus issues with other clients. But historically consensus issues had happened. In that case we start working on a hotfix immediately and release it within hours time. If you need your node to be operational ASAP and can't wait for hotfix you do have an option to achieve that. Nethermind node allows you to FastSync to recent blocks and state. When node does fast sync it can skip over processing problematic blocks. In order to be able to fast sync we need SyncConfig.FastSync to be set to 'true'. You also need to set SyncConfig.FastSyncCatchUpHeightDelta to a value lower than how far your node is behind the chain. SyncConfig.FastSyncCatchUpHeightDelta is the minimum delta between current chain height and node head height when node can switch from full sync (block processing) to fast sync. By default its set to 1024. After setting those values and restarting node, the node will download block headers, bodies (if SyncConfig.DownloadBodiesInFastSync is 'true'), receipts (if SyncConfig.DownloadReceiptsInFastSync is 'true') and current state. After that it will resume processing from new head block. Please note that the historical state for skipped blocks might not be available. This can cause some JSON RPC calls on those historical state not to work - same situation as if this blocks state was pruned.
```

Comment:
'node head height' -> 'chain head block number'

Reply:
'delta' -> 'difference'

Reply:
'its set' -> 'it is set'

Reply:
user has no knowledge whether they can set it to 0 and may end up setting it to 0 -> maybe give them an exmple and tell them no to set it less than 32

Reply:
'those historical state' -> 'the historical state'

Reply:
'as if this blocks' -> 'as if these blocks'<---one more quote'

Reply:
Users need to know that this may take up to 2 hours to skip over the blocks due to the way the state sync fills the gaps. (it all depends on how many blocks they need to skip)

Reply:
added and fixed all